source/css/patternfly.css

```diff
@@ -6092,21 +6092,73 @@ tbody.collapse.in {
       display: inline-block;
       width: 100%; } }
 
-.card {
-  box-shadow: 0 0.1em 0.2em rgba(3, 3, 3, 0.1); }
-
 .pf-card {
   border: none; }
   .pf-card .card-header {
     background: #fff;
-    border-bottom-color: #d1d1d1; }
+    border-bottom-color: #d1d1d1;
+    padding: 1.5rem; }
+    .pf-card .card-header .card-title {
+      margin: 0; }
+    .pf-card .card-header .dropdown {
+      margin: -0.5rem 0; }
   .pf-card .card-footer {
     background: #fafafa;
-    border-top-color: #d1d1d1; }
+    border-top-color: #d1d1d1;
+    padding: 1rem 1.5rem; }
+    .pf-card .card-footer p {
+      margin: 0.25rem 0; }
+  .pf-card .card-block {
+    padding: 1.5rem; }
+  .pf-card .card-title {
+    overflow: hidden;
+    text-overflow: ellipsis; }
+  .pf-card .dropdown.pf-dropdown__kebab {
+    margin: -0.5rem; }
+  .pf-card::after {
+    box-shadow: 0 0.1em 0.2em rgba(3, 3, 3, 0.1);
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    content: "";
+    pointer-events: none; }
+
+.pf-card__top-element {
```

Comment:
I know @srambach hates them, but maybe we could use utility classes here? lets talk about it tomorrow.